rpi-update

```diff
@@ -296,13 +296,27 @@ function noobs_fix {
 
 }
 
+echo " *** Raspberry Pi firmware updater by Hexxeh, enhanced by AndrewS and Dom"
+echo
+echo " *** This script cannot uninstall any firmware automatically, only install"
+echo " *** different versions onto the system. Use with caution. For inexperienced"
+echo " *** users it is recommended to use their distro-provided packages (e.g."
+echo " *** 'raspberrypi-bootloader' or 'rasberrypi-bootloader-nokernel' along with a"
+echo " *** kernel image like 'linux-image-rpi' on raspbian) and NOT use rpi-update to"
+echo " *** avoid conflicts."
+read -p "Are you sure you want to continue? [yn]" -n 1 -r
+echo
+if [[ ! $REPLY =~ ^[Yy]$ ]]
```

Comment:
@dragetd IMHO a global env-var that can be used to override this `read` check would be more consistent with how the rest of the rpi-update script works.

@pelwell Funny you should mention that... #210  ;-)